R/produce_source_cmh_tests.R

```diff
@@ -0,0 +1,37 @@
+#' Source Extract Tests
+#'
+#' @description Produce a set of tests which can be used by the CMH extract
+#'
+#' This will produce counts of various demographics.
+#' It will also produce various summary statistics for episode date variables.
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path()])
+#'
+#' @return a dataframe with a count of each flag
+#' from [calculate_measures()]
+#' @export
+#'
+#' @family extract test functions
+#' @seealso [calculate_measures()]
+produce_source_cmh_tests <- function(data) {
+  test_flags <- data %>%
+    # create test flags
+    create_demog_test_flags() %>%
+    create_hb_test_flags(hb_var = hbrescode) %>%
```

Comment:
```suggestion
    create_hb_test_flags(hb_var = hbrescode) %>%
    dplyr::mutate(n_episodes = 1) %>%
```
Only thing not captured by the tests here is n_episodes which is in the SPSS code